packages/server/src/utils/types.ts

```diff
@@ -38,12 +38,25 @@ export interface DatabaseConfig {
 	client: DatabaseConfigClient;
 	// For Postgres, this is the actual database name. For SQLite, this is the
 	// path to the SQLite file.
-	name: string;
+	name?: string;
+	connectionString?: string;
 	host?: string;
 	port?: number;
 	user?: string;
 	password?: string;
 	asyncStackTraces?: boolean;
+	sslCaFilePath?: string;
+	sslCertFilePath?: string;
+	sslCertKeyFilePath?: string;
+	rejectUnauthorized?: boolean;
```

Comment:
Is it not possible to specify all these options directly via the connection string?

Reply:
I don't think so, the connection string is a postgres thing not a node-postgres thing; these props are describing how to treat the connection.  I will have a look into the docs but when I have used other libraries (I am a C# dev most of the time) I do not include them in the connection string.

Reply:
Ok, thanks for checking.